blocks/v2-tabbed-carousel/v2-tabbed-carousel.js

```diff
@@ -0,0 +1,255 @@
+import { createElement } from '../../scripts/scripts.js';
+
+const blockName = 'v2-tabbed-carousel';
+
+function stripEmptyTags(main, child) {
+  if (child !== main && child.innerHTML.trim() === '') {
+    const parent = child.parentNode;
+    child.remove();
+    stripEmptyTags(main, parent);
+  }
+}
+
+function buildTabNavigation(tabItems, clickHandler) {
+  const tabNavigation = createElement('ul', `${blockName}__navigation`);
+  const navigationLine = createElement('li', `${blockName}__navigation-line`);
+  let timeout;
+
+  [...tabItems].forEach((tabItem, i) => {
+    const listItem = createElement('li', `${blockName}__navigation-item`);
+    const button = createElement('button');
+    button.addEventListener('click', () => clickHandler.call(this, i));
```

Comment:
`this` for the arrow function is undefined. Why not just call the `clickHandler(tabItems, i)`?